examples/common.yaml

```diff
@@ -132,7 +132,7 @@ openstack::swift::hash_suffix: 'pop-bang'
 
 ######## Nova
 
-openstack::nova::libvirt_type: 'kvm'
+openstack::nova::libvirt_type: 'qemu'
```

Comment:
The puppet-openstack module on stackforge is deprecated.

There is also stackforge/puppet-openstack-cloud but it does not have any vagrant examples so this change would not be applicable.